doc/source/how-to/releasing.rst

```diff
@@ -0,0 +1,280 @@
+``__init__.py`` or ``__version__.py`` your project may contain.
+
+Stash and commit previous changes:
+
+.. code-block:: text
+
+   git add . && git commit -m "Bump version X.Y.0 version"
+
+Tag previous commit by running:
+
+.. code-block:: text
+
+   git tag vX.Y.0
+
+Push the commit and the tag:
+
+.. code-block:: text
+
+   git push -u origin release/X.Y && git push origin vX.Y.0
+
+
+Releasing Patched Versions
+--------------------------
+Patched versions allow you to fix issues discovered in published releases by
+cherry-picking these fixes from ``main`` branch.
+
+To create a patched version, you need to identify first which ``release/X.Y``
+branch it does belong to. Make sure your local ``release/X.Y`` is up to date
+with origin one, as other patched versions may have been introduced before.
+
+.. code-block:: text
+
+   git checkout release/X.Y && git fetch origin release/X.Y && git rebase origin/release/X.Y
+
+If you encounter any issues when running previous command, solve those before
+continuing with the release. 
+
+Now, `cherry-pick <https://git-scm.com/docs/git-cherry-pick>`_ the fix commit
+from ``main`` which solves for the bug. Do not merge changes from main into the
+release branch, always cherry-pick them.
+
+.. code-block:: text
+   
+   git cherry-pick <commit hash>
+
+Make sure your style, tests and documentation checks are passing too.
+
+Increase by one unit the value of ``Z`` in your project version. Stash and
+amend these new changes using:
+
+.. code-block:: text
+
+   git add . && git commit --amend -m "Bump version X.Y.Z version"
+
+Tag previous commit by running:
+
+.. code-block:: text
+
+   git tag vX.Y.Z
+
+Push the commit and the tag:
+
+.. code-block:: text
+
+   git push -u origin release/X.Y && git push origin vX.Y.Z
+
+
+Publishing Artifacts
+--------------------
+When a new version is released, some artifacts are provided with it. In Python,
+these :ref:`Artifacts` are typically the ``Wheel`` and ``Source`` files.
+Documentation in the form of PDF and HTML are also considered artifacts.
+
+.. admonition:: Do not distribute artifacts without approval
+
+   Refer to :ref:`Project Approval` for more information about requesting
+   approval for publishing your project.
+
+
+Publishing to Private PyPI
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are cases in which it is necessary to host and pull packages that are
+not ready to be hosted to the public `PyPI`_. For example, if a PyAnsys library
+requires auto-generated gRPC interface files from an as-of-yet private feature
+or service, this package should be hosted on a private PyPI repository.
+
+ANSYS has a private repository at `PyAnsys PyPI`_, and access is controlled via
+a secret ``Personal Access Token (PAT)``, specified in the GitHub secret
+``PYANSYS_PYPI_PRIVATE_PAT`` which is only available to repositories within the
+`PyAnsys`_.
+
+.. admonition:: A note on secrets access in forked repositories
```

Comment:
Couldn't this just be a .. note:: directive?

Reply:
This admonition renders like a note but with a custom bold title.